scripts/build/build-package-json.js

```diff
@@ -73,6 +73,7 @@ async function buildPackageJson({ files }) {
       "node -e \"assert.equal(require('.').version, require('..').version)\"",
   };
   packageJson.files = files.map(({ output: { file } }) => file).sort();
+  packageJson.types = "./index.d.ts";
```

Comment:
Should we update `exports` field?

Reply:
I tried to use [TypeScript 4.7 exports](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-7.html#packagejson-exports-imports-and-self-referencing), but it did not work. At least in our current structure, this configuration works fine.

Reply:
Can you try this?

[`1d60730`](https://github.com/prettier/prettier/pull/14212/commits/1d6073058b3c6225eb04d551bdd54c625be69063)